src/cmd/shared.go

```diff
@@ -64,3 +66,10 @@ func prettyPrintSecrets(secrets map[string]string) {
 	}
 	fmt.Printf("%s\n\n", string(secretsPretty))
 }
+
+// GetDockerComposeProjectName creates a docker compose project name the same way docker-compose mutates the COMPOSE_PROJECT_NAME env var
+func GetDockerComposeProjectName(appName string) string {
+	reg := regexp.MustCompile("[^a-zA-Z0-9]")
+	replacedStr := reg.ReplaceAllString(appName, "")
+	return strings.ToLower(replacedStr)
+}
```

Comment:
Thoughts on putting this in the `docker/compose` package and naming it `GetProjectName()`?